apps/widget/cypress/e2e/notifications-list.spec.ts

```diff
@@ -206,5 +207,5 @@ describe('Notifications List', function () {
 });
 
 function scrollToBottom() {
-  cy.getByTestId('notifications-scroll-area').get('.infinite-scroll-component').scrollTo('bottom');
+  cy.getByTestId('notifications-scroll-area').scrollTo('bottom', { ensureScrollable: true });
```

Comment:
`ensureScrollable` ensures that the target element is scrollable element, otherwise the test will fail

Reply:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** This is helpful context in the code‏